libfprint/drivers/vfs7552_testing.c

```diff
@@ -0,0 +1,873 @@
+/*
+ * Validity Sensors, Inc. VFS7552 Fingerprint Reader driver for libfprint
+ *                        ID 138a:0091 Validity Sensors, Inc.
+ * Copyright (C) 2013 Mark Harfouche <mark.harfouche@gmail.com>
```

Comment:
2013?

Reply:
Fixed it now. I just used the header of your original file ;)